cloudbuild.yaml

```diff
@@ -0,0 +1,12 @@
+steps:
+  # Install dependencies
+  - name: python
+    entrypoint: python
+    args: ["-m", "pip", "install", ".", "--user"]
+
+  # Generate ambient GCP credentials
+  - name: python
+    entrypoint: python
+    args: ["-m", "id", "--audience", "throwaway"]
+    env:
+      - "GOOGLE_SERVICE_ACCOUNT_NAME=sigstore-python-test@projectsigstore.iam.gserviceaccount.com"
```

Comment:
Maybe a non-issue, but does it make sense to have a different Google service account here?

Reply:
Ah, thanks for the reminder, I need to setup these tests to actually run.